src/components/GameView/GameOverDialog.vue

```diff
@@ -1,15 +1,38 @@
 <template>
   <v-dialog v-model="show" persistent max-width="650">
     <v-card id="game-over-dialog">
-      <v-card-title :data-cy="headingDataAttr">
-        <h1>{{ heading }}</h1>
-      </v-card-title>
-      <v-card-text class="d-flex justify-center">
-        <v-img
-          :src="logoSrc"
-          :data-cy="logoDataAttr"
-          class="logo-image"
-        />
+      <div class="d-flex">
+        <v-card-title :data-cy="headingDataAttr" class="mt-8">
+          <h1>{{ heading }}</h1>
+        </v-card-title>
+        <v-card-text class="d-flex justify-end mr-4 mt-4">
+          <v-img
+            :src="logoSrc"
+            :data-cy="logoDataAttr"
+            class="logo-image"
+          />
+        </v-card-text>
+      </div>
+      <v-card-text v-if="currentMatch" data-cy="match-result-section">
+        Match against {{ opponent.username }}
+        <span>: {{currentMatchWinner ? "Finished" : "In Progress"}}</span>
+      </v-card-text>
+      <v-card-text v-if="currentMatchWinner" data-cy="match-winner-message">
+        You {{ playerWinsMatch ? "won" :"lost" }} your game against {{opponent.username}}
+      </v-card-text>
+      <v-card-text data-cy="match-result-games">
+        <div class="d-flex">
+          <div
+            class="d-flex flex-column align-center"
+            v-for="(gameStatus, i) in matchGameStats"
+            :key="i"
```

Comment:
Key should be more unique, something like this to prevent collisions.
```suggestion
            :key="`${gameStatus}-${i}`"
```